src/Akeneo/Category/front/src/feature/components/templates/AddTemplateAttributeModal.tsx

```diff
@@ -0,0 +1,96 @@
+import {translate, useTranslate} from '@akeneo-pim-community/shared';
+import {useState} from "react";
+import {useCreateAttribute} from "../../hooks/useCreateAttribute";
+import {userContext} from "@akeneo-pim-community/shared/lib/dependencies/user-context";
+import {TextInput} from "akeneo-design-system/lib/components/Input/TextInput/TextInput";
+import {AttributesIllustration, Button, Field, Helper, Link, Modal} from "akeneo-design-system";
+import styled from "styled-components";
+import {useHistory} from 'react-router';
+
+const Content = styled.div`
+  padding-bottom: 20px;
+`;
+const FieldSet = styled.div`
+  & > * {
+    margin-top: 20px;
+  }
+`;
+
+const HelperField = styled(Helper)`
+  width: 100%;
+`;
+
+type Form = { label: string; code: string };
+type FormError = { label: string[]; code: string[] };
+
+type Props = {
+    template_id: string;
```

Comment:
```suggestion
    templateId: string;
```
camelCase for typescript code